frontend/src/components/Map/hooks/usePopupResult.ts

```diff
@@ -6,13 +6,14 @@ import { getTouristicContentPopupResult } from 'modules/touristicContent/connect
 import { useRouter } from 'next/router';
 import { getDefaultLanguage } from 'modules/header/utills';
 import { useQueryCommonDictionaries } from 'modules/dictionaries/api';
-import { getTouristicEventPopupResult } from '../../../modules/touristicEvent/connector';
-import { getOutdoorSitePopupResult } from '../../../modules/outdoorSite/connector';
+import { getTouristicEventPopupResult } from 'modules/touristicEvent/connector';
+import { getOutdoorSitePopupResult } from 'modules/outdoorSite/connector';
+import { getOutdoorCoursePopupResult } from 'modules/outdoorCourse/connector';
 
 export const usePopupResult = (
   id: string,
   shouldFetch: boolean,
-  type: 'TREK' | 'TOURISTIC_CONTENT' | 'OUTDOOR_SITE' | 'TOURISTIC_EVENT' | null,
+  type: 'TREK' | 'TOURISTIC_CONTENT' | 'OUTDOOR_SITE' | 'TOURISTIC_EVENT' | 'OUTDOOR_COURSE' | null,
```

Comment:
Shouldn't that Type be defined in just one place? With a nullable variant? 9 references for that change 🤔

Reply:
Right, i just create a dedicated type for that :+1: